pydicom/util/codify.py

```diff
@@ -270,6 +313,7 @@ def code_dataset(
     # If sequence was end of this dataset, remove the extra blank line
     if len(lines) and lines[-1] == "":
         lines.pop()
+
     # Join all the code lines and return them
     return line_term.join(lines)
 
```

Comment:
Unrelated in `code_file_from_dataset` (can't put the comment there) - the first parameter is documented wrong.

Reply:
Done.  Thanks for noticing that.